tests/tools/time.sh

```diff
@@ -22,6 +22,10 @@ wait_for() {
 	return 1
 }
 
+wait_chunkservers() {
```

Comment:
The condition should not be repeated everywhere again, rather it should be in this function (unless it's not possible?)

```suggestion
wait_if_windows() { # Or windows_wait_chunkservers
    if is_windows_system; then
        sleep 5
    fi
}
```
(Please don't commit directly since it uses spaces)

Also as @rolysr mentions, documentation on why this is needed and why 5 seconds exactly.